.github/workflows/release.yml

```diff
@@ -262,6 +267,61 @@ jobs:
           name: artifacts-dist-manifest
           path: dist-manifest.json
 
+      # Unpack binaries, required for setup-pixi
+      - name: Unpack binaries
+        run: |
+          mkdir -p unpacked-artifacts
+          for artifact in target/distrib/*.tar.gz; do
+            # Extract the base name without the extension
+            base_name=$(basename "$artifact" .tar.gz)
+
+            # Create a directory for the specific artifact
+            mkdir -p unpacked-artifacts/$base_name
+
+            # Extract the artifact contents
+            tar -xvf "$artifact" -C unpacked-artifacts/$base_name
+
+            # Move the pixi binary to the base name
+            mv unpacked-artifacts/$base_name/pixi unpacked-artifacts/$base_name
```

Comment:
I would say that for safety reason maybe we could remove after moving
` unpacked-artifacts/$base_name/pixi` folder? so we accidently don't upload it or something

Reply:
Changed that